src/components/data/FooterData.js

```diff
@@ -33,8 +33,8 @@ const PoliciesItems = [
     link: "#"
   },
   {
-    linkText: "Reasonable Accomodation",
-    title: "Reasonable Accomodation",
+    linkText: "Reasonable Accommodation",
+    title: "Reasonable Accommodation",
```

Comment:
I think it would make sense at this point to remove all of the title data, this will allow you to ensure that all title elements have been removed from the prototype.